lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_perform_step.jl

```diff
@@ -1362,11 +840,21 @@ end
         @.. $(_vec(ks[stage]))=-linres.u
         integrator.stats.nsolve += 1
     end
+    du .= ks[end]
     u .+= ks[end]
 
+    @.. u=uprev + b[1] * ks[1] + b[2] * ks[2] + b[3] * ks[3]
```

Comment:
this is running even if  `b` isn't defined in the tableau.

Reply:
so what changes should be made here ??